src/documents/firstDiscoveryServer.md

```diff
@@ -0,0 +1,365 @@
+                A query parameter can be added to provide the view/ontology that the
+                preferences are to be stored in.
+            </td>
+            <td>
+                "/add-preferences?view=%view"
+            </td>
+        </tr>
+        <tr>
+            <td>
+                <code>"preferencesConfig.authentication.grant_type"</code>
+            </td>
+            <td>
+                The grant type supported by the OAuth2 server.
+            </td>
+            <td>
+                "client_credentials"
+            </td>
+        </tr>
+        <tr>
+            <td>
+                <code>"preferencesConfig.authentication.scope"</code>
+            </td>
+            <td>
+                The level of permissions that are being requested.
+            </td>
+            <td>
+                "add_preferences"
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+### vagrant.json ###
+
+Builds on [environment.json](https://github.com/GPII/first-discovery-server/tree/master/src/config/environment.json) and [oauth2.json](https://github.com/GPII/first-discovery-server/tree/master/src/config/oauth2.json), setting concrete values to configuration of the Oauth2 server that are expected when being run via the Vagrant VM. This is typically only used for development.
+
+(see: [vagrant.json](https://github.com/GPII/first-discovery-server/tree/master/src/config/vagrant.json))
+
+<table>
+    <thead>
+        <tr>
+            <th>Option</th>
+            <th>Description</th>
+            <th>Example</th>
+        <tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>
+                <code>"preferencesConfig.securityServer.port"</code>
+            </td>
+            <td>
+                The port number that the GPII OAuth2 server is hosted at.
+            </td>
+            <td>
+                "8081"
+            </td>
+        </tr>
+        <tr>
+            <td>
+                <code>"preferencesConfig.securityServer.hostname"</code>
+            </td>
+            <td>
+                The hostname that the GPII OAuth2 server is hosted at.
+            </td>
+            <td>
+                "http://10.0.2.2"
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+## REST API ##
+
+<table>
+    <thead>
+        <tr>
+            <th>URL</th>
+            <th>Request</th>
+            <th>Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><code>/user?[view=:view]</code></td>
+            <td><code>POST</code></td>
+            <td>
+                Accepts a set of preferences, in a JSON object, to be stored on the preferences server. For example:
+
+                <pre>
+                    <code>
+{
+  "gpii_firstDiscovery_language": "en-US"
+}
+                    </code>
+                </pre>
+
+                A GPII token will be returned and can be used for retrieving the preferences on a GPII enabled device. For example:
+
```

Comment:
Is this an example of what is returned by the FD server to the FD tool? Because it's more than just a token... or is it what would be retrieved if you use the token to retrieve the prefs